craft_providers/multipass/multipass.py

```diff
@@ -50,10 +50,10 @@ def _run(
         self,
         command: List[str],
         *,
-        check=True,
+        check: bool,
         **kwargs,
     ) -> subprocess.CompletedProcess:
-        """Execute command in instance_name, allowing output to console."""
+        """Execute command on host, allowing output to console."""
```

Comment:
I don understand, from the point of view of the user of the API, either that or "host" is too generic.

Reply:
Fair enough, simplified to `execute multipass command` :D